nixos/modules/services/misc/rsync.nix

```diff
@@ -0,0 +1,153 @@
+{
+  config,
+  lib,
+  pkgs,
+  utils,
+  ...
+}:
+let
+  cfg = config.services.rsync;
+  inherit (lib) types;
+  inherit (utils.systemdUtils.unitOptions) unitOption;
+in
+{
+  options.services.rsync = {
+    package = lib.mkPackageOption pkgs "rsync" { };
```

Comment:
Please add a `enable` attribute like in pretty much all the Nixon modules available.

Reply:
That makes sense. I left it out just to stay consistent with similar modules like `services.borgbackup` or `services.restic`. I could definitely add an `enable` option here, but if I do, it would make sense to consider doing the same for those other modules down the line.

Reply:
Yes, definitely.